core/tests/unit/test_sfn.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Could we create a test where the `GlueTransformStage` is used in a DDK pipeline? Just so we can test these new methods you have created

Reply:
Sure